scripts/test/SANS/algorithm_detail/batch_execution_test.py

```diff
@@ -335,22 +337,49 @@ def test_that_save_workspace_to_file_includes_run_numbers_in_options(self, mock_
         }
         mock_alg_manager.assert_called_once_with("SANSSave", **expected_options)
 
-    @mock.patch("sans.algorithm_detail.batch_execution.create_unmanaged_algorithm")
-    def test_that_non_subtracted_save_workspace_to_file_does_not_include_metadata_in_options(self, mock_alg_manager):
-        ws_name = "wsName"
-        filename = "fileName"
-        additional_run_numbers = {}
-        additional_metadata = {"BackgroundSubtractionWorkspace": "tobesubtracted", "BackgroundSubtractionScaleFactor": "1.25"}
+    @mock.patch("sans.algorithm_detail.batch_execution.get_all_names_to_save")
+    @mock.patch("sans.algorithm_detail.batch_execution.save_workspace_to_file")
+    def test_that_non_subtracted_save_to_file_does_not_include_metadata_in_options(self, mock_save_func, mock_names_func):
+        state = mock.MagicMock()
+        save_info_mock = mock.MagicMock()
+        scaled_bg_mock = mock.MagicMock()
+        scaled_bg_mock.workspace = "scaled_ws"
+        scaled_bg_mock.scale_factor = 0.5
+        file_formats_mock = mock.MagicMock()
+        save_info_mock.file_format = file_formats_mock
+        state.save = save_info_mock
+        state.background_subtraction = scaled_bg_mock
+        reduction_package = mock.MagicMock()
+        reduction_package.state = state
 
-        save_workspace_to_file(ws_name, [], filename, additional_run_numbers, additional_metadata)
+        mock_names_func.return_value = ["unsubbed_ws"]
+        save_to_file([reduction_package], False, {}, {})
+        mock_save_func.assert_called_with("unsubbed_ws", file_formats_mock, "unsubbed_ws", {}, {})
 
-        expected_options = {
-            "InputWorkspace": ws_name,
-            "Filename": filename,
-            "Transmission": "",
-            "TransmissionCan": "",
-        }
-        mock_alg_manager.assert_called_once_with("SANSSave", **expected_options)
+    @mock.patch("sans.algorithm_detail.batch_execution.get_all_names_to_save")
+    @mock.patch("sans.algorithm_detail.batch_execution.save_workspace_to_file")
+    def test_that_subtracted_save_to_file_does_not_includes_metadata_in_options(self, mock_save_func, mock_names_func):
```

Comment:
Was this test name intended to be: `test_that_subtracted_save_to_file_includes_metadata_in_options`?